src/styles/variables.css

```diff
@@ -0,0 +1,20 @@
+@import url('https://fonts.googleapis.com/css?family=Archivo:400,500,600,700&display=swap');
```

Comment:
We can improve the performance of this font import by only using the Latin versions and also remove the 600 font-weight since the design doesn't use that weight. Let me know if you need any help with this improvement.

Reply:
I'll push a commit with this change.